src/index.js

```diff
@@ -0,0 +1,151 @@
+const state = {
+	cityLocation: { lat: 28.5421109, lon: -81.3790304 },
+};
+
+const loadControls = () => {
+	state.tempValueEl = document.getElementById('tempValue');
+	state.increaseTempControl = document.getElementById('increaseTempControl');
+	state.decreaseTempControl = document.getElementById('decreaseTempControl');
+	state.tempValue = parseInt(state.tempValueEl.textContent);
+	state.gardenLandscape = document.getElementById('landscape');
+	state.gardenLandscapeValue = state.gardenLandscape.textContent;
+	state.cityLocationEl = document.getElementById('cityNameInput');
+	state.skyDisplay = document.getElementById('sky');
+	state.skyDisplayValue = state.skyDisplay.textContent;
+};
+
+const increaseTemp = () => {
+	state.tempValue += 1;
+	state.tempValueEl.textContent = `${state.tempValue}`;
+	changeColorAndLandscape();
+};
+
+const decreaseTemp = () => {
+	state.tempValue -= 1;
+	state.tempValueEl.textContent = `${state.tempValue}`;
+	changeColorAndLandscape();
+};
+
+const changeColorAndLandscape = () => {
+	let temp = state.tempValue;
```

Comment:
It looks like this project is using tabs instead of spaces. I recommend looking at how to change this in VS Code, tabs are not handled the same by various text editing and formatting tools and can cause unexpected weirdness for other folks who pull down or review the code.